dns_entry_checker/test/test_dns_entry_checker.py

```diff
@@ -0,0 +1,193 @@
+import dns_entry_checker
+import unittest
+from unittest import mock
+from unittest.mock import MagicMock, patch, NonCallableMock, call
+from parameterized import parameterized
+from dns_entry_checker import (
+    create_client,
+    ssh_command,
+    find_ip_dns_pair,
+    check_ip_dns_mismatch,
+    check_missing_ips,
+)
+
+
+class DNSEntryCheckerTests(unittest.TestCase):
+    @patch("dns_entry_checker.paramiko")
+    def test_create_client(self, mock_paramiko):
+        host = "test.aquilon.gridpp.rl.ac.uk"
+        user = "test.abc12345"
+        password = "test.A4gdfGs"
+
+        mock_client = MagicMock()
+        mock_paramiko.client.SSHClient.return_value = mock_client
+        mock_paramiko.client.set_missing_host_key_policy = MagicMock()
+        mock_paramiko.AutoAddPolicy = NonCallableMock()
+        mock_paramiko.connect = MagicMock()
+
+        client = create_client(host, user, password)
+
+        client.connect.assert_called_once_with(
```

Comment:
Add this line to test that set_missing_host_key_policy function was called property
```suggestion
        client.set_missing_host_key_policy.assert_called_once_with(mock_paramiko.AutoAddPolicy)
        client.connect.assert_called_once_with(
```